eclair-core/src/main/scala/fr/acinq/eclair/wire/LightningMessageCodecs.scala

```diff
@@ -326,6 +364,11 @@ object LightningMessageCodecs {
     .typecase(263, queryChannelRangeCodec)
     .typecase(264, replyChannelRangeCodec)
     .typecase(265, gossipTimestampFilterCodec)
+  //
+    .typecase(1487, provide(QuerySupportedPlugins))
```

Comment:
these are magic odd numbers, should they be picked on some specific principle?

Reply:
Yes, as per https://github.com/lightningnetwork/lightning-rfc/blob/master/01-messaging.md#lightning-message-format :

> Custom (types 32768-65535): experimental and application-specific messages